docs/map-making/upgrading-1-8-maps.md

```diff
@@ -1,14 +1,22 @@
 A list of (case sensitive) changes, to upgrade a map from a TripleA version to another one.
 
-When changes are required, all occurrences of the code before the colon must be substituted with the code after the colon (if there is no code after the colon, the code must be substituted with nothing (deleted)). In any code, the "\*" symbol means whatever (the change or deletion is to be applied to every given code having whatever character or characters in place of the "\*").
+When changes are required, all occurrences of the code before the `>>` must be substituted
+with the code after the `>>` (if there is no code after the `>>`, the code must be substituted
+with nothing (deleted)). In any code, the "\*" symbol means whatever (the change or deletion is
+to be applied to every given code having whatever character or characters in place of the "\*").
 
```

Comment:
You have removed all cases in which "there is no code after the `>>`", to mean deletions, so that's now a pointless thing to say.